api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/VisitsServiceClientIntegrationTest.java

```diff
@@ -133,6 +136,116 @@ void createVisitForPet_Valid() throws JsonProcessingException {
                 .verifyComplete();
 
     }
+
+    //DuplicateTime Exception Test
```

Comment:
[Nitpic]: since the test has a good name, I believe this comment saying what the test is, is redundant.
```suggestion
```